docker/README.md

```diff
@@ -164,91 +233,106 @@ In order to sign packages, you need to generate and place GPG keys in the right
     6. Email address (Will become part of the key and uid)
     7. Comment (Will become part of the key)
     8. O for Okay
-    9. Enter a very secure password, make sure you will not forget it
+    9. Enter a secure password, make sure you will not forget it
     10. Generate Entropy!
     
-2. gpg -a --export > ./docker/public.key
-
-3. gpg -a --export-secret-keys > ./docker/signing.key
+    export --export-secret-key '<key ID>!' >
+    
+e.g. `export --export-secret-key '547CF247!' > ./docker/git-lfs_centos_7.key`
```

Comment:
Of course, this doesn't work on the mac :)

    $ gpg -a --export-secret-keys <key ID>